Sources/Dependencies/DependencyKey.swift

```diff
@@ -118,12 +118,9 @@ public protocol DependencyKey: TestDependencyKey {
 /// return a default value suitable for Xcode previews, or the ``testValue``, if left unimplemented.
 ///
 /// See ``DependencyKey`` to define a static, default value for the live application.
-public protocol TestDependencyKey {
+public protocol TestDependencyKey: Sendable where Value: Sendable {
   /// The associated type representing the type of the dependency key's value.
-  associatedtype Value = Self
-
-  // NB: This associated type should be constrained to `Sendable` when this bug is fixed:
-  //     https://github.com/apple/swift/issues/60649
+	associatedtype Value = Self
```

Comment:
Looks like the fix landed in 5.7.1: https://github.com/apple/swift/blob/swift-5.7.1-RELEASE/lib/Sema/CSSimplify.cpp#L3862-L3868

```suggestion
  #if swift(>=5.7.1)
  	associatedtype Value: Sendable = Self
  #else
    // NB: Can't constrain to `Sendable` on earlier Swift versions due to this bug:
    //     https://github.com/apple/swift/issues/60649
    associatedtype Value = Self
  #endif
```